zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestClusterBuilder.java

```diff
@@ -33,7 +33,8 @@ public final class TestClusterBuilder {
 
   private Consumer<TestApplication<?>> nodeConfig = node -> {};
   private BiConsumer<MemberId, TestStandaloneBroker> brokerConfig = (id, broker) -> {};
-  private BiConsumer<MemberId, TestGateway<?>> gatewayConfig = (id, gateway) -> {};
+  private BiConsumer<MemberId, TestStandaloneGateway> gatewayConfig = (id, gateway) -> {};
+  private boolean allowUnauthenticatedAccess = false;
```

Comment:
🔧 I would actually argue for making the the default true for TestStandaloneBroker? Having no auth for a test setup seems acceptable and actually convenient and it would save a lot of changes where you now needed to add `withUnauthenticatedAccess`

Reply:
There are a few tests which either don't require access or which actually test authentication. I'd expect `TestStandaloneBroker` (and other TestSpringApplications) to run with the default configuration by default and explicitly state in tests when non-default configuration is required. 

Being explicit also helps with future changes. For example, it's likely that in the future there will be features that don't work without authentication (e.g. audit logs or even just tracking the creator of an entity).